forms/uds/a1/form_a1_ivp_error_checks_p.csv

```diff
@@ -0,0 +1,52 @@
+GENWOMAN,
+GENTRMAN,
+GENTRWOMAN, GENNONBI,
+GENTWOSPIR,
+GENOTH,
+GENDKN,
+GENNOANS",No,No,
+a1-ivp-p-1003,1003,Alert,a1,I,GENMAN,Plausibility,"If one gender is known, cannot check No answer","If  GENMAN, GENWOMAN, GENTRMAN, GENTRWOMAN, GENNONBI, GENTWOSPIR, GENOTH, or GENDKN=1 then GENNOANS cannot equal 1","If  Q3a. GENMAN, Q3b. GENWOMAN, Q3c. GENTRMAN, Q3d. GENTRWOMAN, Q3e. GENNONBI, Q3f. GENTWOSPIR, Q3g. GENOTH, or Q3h. GENDKN=1 then Q3i. GENNOANS cannot equal 1",IF (GENMAN=1 or GENWOMAN=1 or GENTRMAN=1 or GENTRWOMAN=1 or GENNONBI=1 or GENTWOSPIR=1 or GENOTH=1 or GENDKN=1) and GENNOANS=1,,"GENMAN,
+GENWOMAN,
+GENTRMAN,
+GENTRWOMAN, GENNONBI,
+GENTWOSPIR,
+GENOTH,
+GENDKN,
+GENNOANS",No,No,
+a1-ivp-p-1004,1004,Alert,a1,I,SEXORNGAY,Plausibility,"If one sexual orientation is known, cannot check Don't know","If  SEXORNGAY, SEXORNHET, SEXORNBI, SEXORNTWOS, SEXORNOTH, or SEXORNNOAN=1 then SEXORNDNK cannot equal 1","If Q6a. SEXORNGAY, Q6b. SEXORNHET, Q6c. SEXORNBI, Q6d. SEXORNTWOS, Q6e. SEXORNOTH, or Q6g. SEXORNNOAN=1 then Q6f. SEXORNDNK cannot equal 1",IF (SEXORNGAY=1 or SEXORNHET=1 or SEXORNBI=1 or SEXORNTWOS=1 or SEXORNOTH=1 or SEXORNNOAN=1) and SEXORNDNK=1,,"SEXORNGAY,
+SEXORNHET,
+SEXORNBI,
+SEXORNTWOS,
+SEXORNOTH,
+SEXORNDNK,
+SEXORNNOAN",No,No,
+a1-ivp-p-1005,1005,Alert,a1,I,SEXORNGAY,Plausibility,"If one sexual orientation is known, cannot check No answer","If  SEXORNGAY, SEXORNHET, SEXORNBI, SEXORNTWOS, SEXORNOTH, or SEXORNDNK=1 then SEXORNNOAN cannot equal 1","If Q6a. SEXORNGAY, Q6b. SEXORNHET, Q6c. SEXORNBI, Q6d. SEXORNTWOS, Q6e. SEXORNOTH, or Q6f. SEXORNDNK=1 then Q6g. SEXORNNOAN cannot equal 1",IF (SEXORNGAY=1 or SEXORNHET=1 or SEXORNBI=1 or SEXORNTWOS=1 or SEXORNOTH=1 or SEXORNDNK=1) and SEXORNNOAN=1,,"SEXORNGAY,
+SEXORNHET,
+SEXORNBI,
+SEXORNTWOS,
+SEXORNOTH,
+SEXORNDNK,
+SEXORNNOAN",No,No,
+a1-ivp-p-1006,1006,Alert,a1,I,EXPANCEST,Plausibility,"If one main reason for the experience is known, cannot check No answer","If  EXPANCEST, EXPGENDER, EXPRACE, EXPAGE, EXPRELIG, EXPHEIGHT, EXPWEIGHT, EXPAPPEAR, EXPSEXORN, EXPEDUCINC, EXPDISAB, or EXPSKIN=1 then EXPNOANS cannot equal 1","If Q22a. EXPANCEST, Q22b. EXPGENDER, Q22c. EXPRACE, Q22d. EXPAGE, Q22e. EXPRELIG, Q22f. EXPHEIGHT, Q22g. EXPWEIGHT, Q22h. EXPAPPEAR, Q22i. EXPSEXORN, Q22j. EXPEDUCINC, Q22k. EXPDISAB, or Q22l. EXPSKIN=1, then Q22m. EXPNOANS cannot equal 1",IF (EXPANCEST=1 or EXPGENDER=1 or EXPRACE=1 or EXPAGE=1 or EXPRELIG=1 or EXPHEIGHT=1 or EXPWEIGHT=1 or EXPAPPEAR=1 or EXPSEXORN=1 or EXPEDUCINC=1 or EXPDISAB=1 or EXPSKIN=1) and EXPNOANS=1,,"EXPANCEST,
```

Comment:
Is this a check for A1 form, can't find these variables in questions_and_vars sheet? And Q22 in A1 is a different question than what is mentioned here.

Reply:
Oops, yes these were originally in A1 and got moved to A1a!